packages/tradershub/src/flows/FirstSignup/CitizenshipModal/CitizenshipModal.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+import { useFormikContext } from 'formik';
+import { Modal } from '@/components';
+import { useClientCountry, useResidenceList } from '@deriv/api';
+import { LabelPairedChevronDownMdRegularIcon } from '@deriv/quill-icons';
+import { Button, Checkbox, Dropdown, Text } from '@deriv-com/ui';
+import { TFirstSignupFormValues } from '../FirstSignupWrapper/FirstSignupWrapper';
+
+type TCitizenshipModal = {
+    onClickNext: VoidFunction;
+};
+
+const CitizenshipModal = ({ onClickNext }: TCitizenshipModal) => {
+    const { data: residenceList, isLoading: residenceListLoading } = useResidenceList();
+    const { data: clientCountry, isLoading: clientCountryLoading } = useClientCountry();
+    const [isCheckBoxChecked, setIsCheckBoxChecked] = useState(false);
+    const { values, setFieldValue } = useFormikContext<TFirstSignupFormValues>();
+    const isBrazil = values.country === 'br';
+
+    useEffect(() => {
+        if (residenceList?.length && clientCountry && values.country === '') {
+            setFieldValue('country', clientCountry);
+        }
+    }, [clientCountry, setFieldValue, residenceList, values.country]);
+
+    // Add <Loading /> here later when it's created
+    if (clientCountryLoading && residenceListLoading) return null;
+
+    return (
+        <Modal className='h-full rounded-default max-w-[328px] md:max-w-[440px]'>
+            <div className='flex flex-col max-w-[296x] md:max-w-[392px] p-16 space-y-16 md:space-y-24 md:p-24'>
```

Comment:
for the screens which  we don't have any consideration for tablet, we will consider it same as mobile, so our breakpoint should be `lg` instead of `md`